src/sync/sys/windows/net.rs

```diff
@@ -321,9 +345,14 @@ impl ClientConnection {
         opts.read(true)
             .write(true)
             .custom_flags(FILE_FLAG_OVERLAPPED);
-        let file = opts.open(self.address.as_str());
-
-        return PipeConnection::new(file.unwrap().into_raw_handle() as isize)
+        match opts.open(self.address.as_str()) {
+            Ok(file) => {
+                return PipeConnection::new(file.into_raw_handle() as isize)
+            }
+            Err(e) => {
+                return Err(Error::Windows(e.raw_os_error().unwrap()))
```

Comment:
Won't this `unwrap` be a problem? I didn't delve into it, just a reminder

Reply:
This was actually from https://github.com/containerd/ttrpc-rust/pull/182/files#diff-fe6e1d087299477f5c1212ceaef02f4b9982bc886ce16361b582f160ab0c614cR329 but needed it here to get the tests to pass, will need to rebase it after that one merges.  

But this is a great point, I keep forgetting that `.unwrap` panics, it is so easy to use (and I didn't know that when I first started on this work 😬).  Since this is several places throughout the Windows file here, I think I will clean it up in a separate PR if that works.